ERCS/erc-7573.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7573
+title: Conditional-upon-Transfer-Decryption for DvP
+description: A Proposal for a Lean and Functional Delivery versus Payment
+author: Christian Fries (@cfries), Peter Kohl-Landgraf (@pekola)
+discussions-to: https://ethereum-magicians.org/t/erc-7573-conditional-upon-transfer-decryption-for-delivery-versus-payment/17232
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-12-05
+---
+
+## Abstract
+
+The interfaces model the functional transaction scheme to establish a secure delivery-versus-payment
```

Comment:
You should define what a "delivery-versus-payment" is somewhere early on in your abstract. Not everyone reading this document will be as familiar as you are with the topic.

Reply:
Thank you. A short definition has been added.